pkg/utils/condition/condition.go

```diff
@@ -97,6 +99,21 @@ const (
 	AllWorkAvailableReason = "AllWorkAreAvailable"
 )
 
+// IgnoreConditionLTTAndMessageFields is a cmpopts.IgnoreFields that ignores the LastTransitionTime and Message fields
```

Comment:
we cannot put the test utils into a production package. You can move to test/utils folder.